### PR TITLE
Makefile typo fixes

### PR DESCRIPTION
**Pull request by realloc**
SWAGGER_URL must be defined externally and fall back to default only when not set.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -131,7 +131,7 @@ docker/generate-server:
 		quay.io/goswagger/swagger:$(SWAGGER_VERSION) generate server \
 			-t gen -f ./spec/rest.yaml --exclude-main \
 			-A neofs-rest-gw -P models.Principal \
-            -C templates/server-config.yaml --template-dir templates
+						-C templates/server-config.yaml --template-dir templates
```
Comment (KirillovDenis) [inline Makefile:134]: Maybe we should format like this:
```suggestion
			-C templates/server-config.yaml --template-dir templates
```

Comment (KirillovDenis) [inline Makefile:134]: Why do we need additional indents here?

Comment (KirillovDenis) [reply Makefile:134]: I would align with previous line

Comment (realloc) [reply Makefile]: fixed

**Review by KirillovDenis — APPROVED** [review]

**Pull request merged**